### DISCUSSION TITLE
Brand new install of pyrosimple questions

### DISCUSSION THREAD

**Discussion by govardha** [kannibalox/pyrosimple#47] [Q&A] [answered]
Hello,,

I had to start afresh with my install and I am noticing some weird issues.  I have done the following
1. Install Debian 12
2. Install swizzin which further installed rtorrent for me with a panel.
3. I upgraded the native version of rtorrent with the version recommended by @kannibalox.  The rtorrent install spins up properly.
4. I created a venv environment and further installed pyrosimple & pyrotorque.  Now comes the interesting stuff.
5. I created the default config using the create-config switch.
6. When I execute the command shown below.

`rtxmlrpc system.hostname`

I get contents of my default ~/.rtorrent.rc is echoed out followed by the hostname

```
throttle.min_peers.seed.set -1
{}
trackers.use_udp.set yes
{}
...
mynodename
```

What am I doing wrong here?  I rolled back to v2.10.0 but still get the same.   I am not sure what I am doing wrong.  Any thoughts appreciated.

Thanks!
g

**Comment by kannibalox** [comment, marked as answer]
Give the v2.10.2 a shot, there were some debugging statements left in accidentally.

**Comment by govardha** [comment]
Thank you that worked!
